src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -1,15 +1,19 @@
 package mate.academy.service;
 
-import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger();
+
     @Override
-    public User login(String login, String password) throws AuthenticationException {
+    public User login(String login, String password) {
         //TODO: add corresponding log message about method login was called
```

Comment:
The TODO comment should be removed before submitting the solution, as per the checklist requirement: 'Let's remove all TODO comments before submitting solution.'